docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,415 @@
+# User Guide to Integrate a New Platform into the OpenROAD Flow
+
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Overview](#Overview)
+- [Prerequisites](#Prerequisites)
+- [Adding a new Platform to OpenROAD](#adding-a-new-platform-to-openroad)
+	- [Setup](#Setup)
+		- [Makefile](#Makefile)
+		- [Platform Directory](#platform-directory)
+		- [Design Directory](#design-directory)
+	- [Platform Configuration](#platform-configuration)
+	- [Design Configuration](#design-configuration)
+		- [config.mk](#configmk)
+		- [constraint.sdc](#constraintsdc)
+		- [Liberty, LEF, and GDS Files](#liberty-lef-and-gds-files)
+	- [Behavioral Models](#behavioral-models)
+		- [Gated Clock](#gated-clock)
+		- [Latch](#latch)
+	- [FastRoute Configuration](#fastroute-configuration)
+	- [Metal Tracks Configuration](#metal-tracks-configuration)
+	- [PDN Configuration](#pdn-configuration)
+	- [Tapcell Configuration](#tapcell-configuration)
+	- [setRC Configuration](#setrc-configuration)
+	- [KLayout](#klayout)
+		- [KLayout properties file](#klayout-properties-file)
+		- [KLayout tech file](#klayout-tech-file)
+- [Validating the New Platform](#validating-the-new-platform)
+- [Authors/Contributors](#authorscontributors)
+
+
+# Overview
+
+This document is a guide for foundry and third party IP providers to easily integrate and test a new technology in to the OpenROAD RTL to GDS flow. OpenROAD allows you to integrate any PDK (Process Design Kit) for any feature size and implement a fully open-sourced RTL-GDSII flow (synthesizable Verilog to merged GDSII). The OpenROAD flow has been validated for feature sizes down to 7nm and used to design and tapeout over 100 ASIC and SoCs to date.
```

Comment:
I don't think we're at the point where foundries are concerned with OpenROAD. This is more targeted at end users to set up a new platform.

Reply:
Even though the end goal of OpenROAD is to be targeted to foundries/3rd parties? At this point, are we more targeted to academia?

Reply:
It's certainly an aspiration, but we don't want to scare a user away because they want to port a platform, but they don't fit into either of those categories. All of the setup done by the OpenROAD team doesn't fit into that category.